### PR TITLE
Feature/etherscan integration

### PR DESCRIPTION
**Pull request by josepablofm78**
- Etherscan integration to enrich raw logs
- Transform logs in a flat format

### CODE REVIEW

File: src/main/java/io/keyko/monitoring/config/StreamerConfig.java [hunk 1]
```diff
@@ -8,41 +8,63 @@
 
 public class StreamerConfig {
 
+  private static final String APPLICATIONID = "applicationId";
```
Comment (aaitor) [inline src/main/java/io/keyko/monitoring/config/StreamerConfig.java]: APPLICATION_ID

File: src/main/java/io/keyko/monitoring/config/StreamerConfig.java [hunk 2]
```diff
@@ -129,4 +151,59 @@ public void setAllTopics(Collection<Object> values) {
     this.topicList = list;
   }
 
+  public String getEtherscanUrl() {
```
Comment (aaitor) [inline src/main/java/io/keyko/monitoring/config/StreamerConfig.java]: maybe we can use `lombok` like in the agent to save us of the big POJO classes?

File: src/main/java/io/keyko/monitoring/examples/basic/BasicStreamManager.java [hunk 3]
```diff
@@ -18,7 +24,19 @@ public BasicStreamManager(StreamerConfig config) {
...
+    String contractAbiParams = configuration.getEtherscanGetContract();
+    String url = ethScan.concat(contractAbiParams);
+
+    KStream<String, EventRecord> eventFromLogStream = Transformations.transformLogToEvent(logStream,
+                                                                          url,
+                                                                          configuration.getEtherscanApikey(),
+                                                                          configuration.getEtherscanSendNotMatchToTopic(),
+                                                                          configuration.getLogNotMatchErrorTopic());
+
+    eventFromLogStream.to("w3m-events-from-log", Produced.with(Serdes.String(), Web3MonitoringSerdes.getEventSerde()));
```
Comment (aaitor) [inline src/main/java/io/keyko/monitoring/examples/basic/BasicStreamManager.java]: I guess the topic should come from configuration

Comment (josepablofm78) [reply src/main/java/io/keyko/monitoring/examples/basic/BasicStreamManager.java]: it's just an example of how to use it, not part of the framework

File: src/main/java/io/keyko/monitoring/config/StreamerConfig.java (hunk 2 above)
Comment (josepablofm78) [reply src/main/java/io/keyko/monitoring/config/StreamerConfig.java]: could be interesting for next releases

**Pull request merged**